### ISSUE TITLE
login or create an user

### ISSUE THREAD

**Issue by gismo07** [khaxis/plynx#25]
Hi,

I just cloned the repo and started the service using docker-compose.
But how do I login or create a new user?

**Comment by khaxis** [comment]
Hi,

TL;DR Please go to http://localhost:3001/ in your browser.

Did you start it directly with `docker-compose` file or running `make up`? The later one should have opened a tab in your browser where you get new credentials.

There there are two modes:
1. (default) Single user. Home page automatically log you in. Currently working only from `/`
2. Multiple users. Similarly to airflow, you can do it only using CLI. `$ plynx users` command line should do it. Note that you need to `ssh` to a container with environment where plynx CLI is installed and credential are right. Backend for example.

**Comment by gismo07** [comment]
Yes I've allready visited localhost:3001.

There is one error message in the console:

`OPTIONS http://localhost:5000/plynx/api/v0/token net::ERR_CONNECTION_REFUSED`

**Comment by khaxis** [comment]
Unfortunately I could not replicate it. Are you using `make up` command to run it?

Do you have the same error as here? https://github.com/khaxis/plynx/issues/23 It was fixed in https://github.com/khaxis/plynx/commit/37c88a56f243b8ec8d46dc3faa6d5d5481c824d1

**Comment by khaxis** [comment]
If not can you please share `make up` output and everything else in console please? It would be very helpful

**Comment by gismo07** [comment]
Hi sorry for my late response. 
I figured it out: the backend was down. Just crashed after few seconds.

Im testing on windows. After changing the volumes to virtual ones, everything is working.

If wanted I can share the adapted docker-compose.yaml.

**Comment by khaxis** [comment]
Cool, thanks! Sure, what is the fix?
